### ISSUE TITLE
RX Refill - Expand Touch Area to Get to Prescription Details

### ISSUE THREAD

**Issue by bischoffa** [department-of-veterans-affairs/va-mobile-app#4616]
## Description 
As a Veteran using VA Health and Benefits App, I want to be able to click on a bigger space on the Medicine Module in Prescriptions, so that its more accommodating to Veterans with Accessibility needs. 

As part of this story we want to:   
- Expand the space in which the Veteran can click and get to the prescription details 

## Acceptance Criteria
  

-  [ ] AC 1
GIVEN Veterans in the VA Health and Benefit App's Prescription section
WHEN clicking the med name
THEN taken to prescription details 

-  [ ] AC 2
GIVEN Veterans in the VA Health and Benefit App's Prescription section
WHEN clicking around prescription details area
THEN taken to prescription details 

**Accessibility Requirements**
- [ ] Setting should be read by the screen reader
- [ ] If anything is being conveyed visually only (buttons, menu items, list, tables, images, headings, forms, etc) the
component type needs to be announced by the screenreader. (Include name (title), role, value)
Each page should support text resizing based on the OS font scaling settings without loss of content, content overlap,
or horizontal scroll.
- [ ] Each touch target must be at minimum 44px by 44px
- [ ] All elements on a page should be able to be traversed by an auxiliary keyboard. As a user uses the keyboard to navigate a page, its focus element should be clear.
- [ ] The app needs to support operability by tap (no gestures) or voice control only. In cases where gestures occur, e.g. scroll, ensure these simple gestures are operable through the OS’ built-in capabilities.

## Notes & Open Questions
 - Finding from 1/26 UAT (Keith), he indicated that he intuitively clicks the name of the medicine or next to it to proceed to more details before seeing the small area that says get more prescription details. Referenced from an accessibility perspective it may be too small of an area to click.  
- Will need to validate and prioritize 

- Dependencies/Roadblocks:
    - Any internal/external dependencies? No
    - Test accounts needed? Yes
    - Does this require QA? Yes
    
- Dev Notes: 

## Ticket Checklist

- [ ] Acceptance criteria defined
- [x] Labels added (front-end, back-end, feature)
- [x] Linked to an Epic

**Comment by b-rocha** [comment]
Closing due to the rationale noted above by Therese on 2/1/2023.
